addons/sys_data/fnc_deserialize.sqf

```diff
@@ -0,0 +1,29 @@
+/*
+ * Author: ACRE2Team
+ * SHORT DESCRIPTION
+ *
+ * Arguments:
+ * 0: ARGUMENT ONE <TYPE>
+ * 1: ARGUMENT TWO <TYPE>
+ *
+ * Return Value:
+ * RETURN VALUE <TYPE>
+ *
+ * Example:
+ * [ARGUMENTS] call acre_sys_data_fnc_serialize;
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private ["_ret"];
+if (IS_SERIALIZEDHASH(_this)) then {
```

Comment:
This macro is undefined. The definition was previously in `sys_data/XEH_preInit.sqf`
Definition should be
`#define IS_SERIALIZEDHASH(array) (IS_ARRAY(array) && {(count array) > 0} && {IS_STRING((array select 0))} && {(array select 0) == "ACRE_HASH"})`

Alternative is to move into the `sys_data/script_component.hpp` for use throughout the addon.